src/main/resources/vm/script/datamanagement/refresh.vm

```diff
@@ -0,0 +1,97 @@
+## refresh.vm
+
+
+shanoir_token_location="${PWD}/cache/SHANOIR_TOKEN.txt"
+shanoir_refresh_token_location="${PWD}/cache/SHANOIR_REFRESH_TOKEN.txt"
+refreshingJobStarted=false
+
+REFRESH_PID=""
+
+#
+# this is a background process to refresh shanoir token
+#
+function refresh_token {
+    touch $shanoir_token_location
+    touch $shanoir_refresh_token_location
```

Comment:
As the `refresh_token` function will be launch in a background shell, can it access the `$shanoir_token_location` variable ? I made a quick test and I was not the case, but I may have missed something

Reply:
yes, it can read the variable, but it can't change its value. it's like they become local variables inside that sub-shell